src/list-grid.scss

```diff
@@ -342,4 +350,10 @@ $block: #{$fd-namespace}-grid-list;
       }
     }
   }
+
+  &__btn-navigation {
+    @include fd-rtl() {
+      transform: rotate(180deg);
+    }
```

Comment:
this is not used anywhere. Why do you need it?

Reply:
If it's a class needed in fund-ngx you need to add an example in styles so we have a visual test to avoid bugs in future. Also, it's a new class and needs to be added in the Breaking changes wiki.

Reply:
added example and breaking change information.